test/unit/util/AsyncHandler.test.ts

```diff
@@ -1,7 +1,17 @@
-import type { AsyncHandler } from '../../../src/util/AsyncHandler';
+import { AsyncHandler } from '../../../src/util/AsyncHandler';
 import { StaticAsyncHandler } from '../../util/StaticAsyncHandler';
 
 describe('An AsyncHandler', (): void => {
+  it('supports any input by default.', async(): Promise<void> => {
+    class EmptyHandler<T> extends AsyncHandler<T, null> {
+      public async handle(): Promise<null> {
+        return null;
+      }
+    }
+    const handler = new EmptyHandler<string>();
+    await expect(handler.canHandle('test')).resolves.toBe(undefined);
```

Comment:
Just to be consistent with the other tests, jest also has an `toBeUndefined()` function.

Reply:
Absolutely, will amend and merge.